modules/understand-admission-controller-enforcement.adoc

```diff
@@ -5,23 +5,34 @@
 [id="understand-admission-controller-enforcement_{context}"]
 = Understanding admission controller enforcement
 
-If you intend to use admission controller enforcement, consider the following:
+[role="_abstract"]
 
-* *API latency*: Using admission controller enforcement increases Kubernetes or {ocp} API latency because it involves additional API validation requests.
+{product-title-short} uses an admission controller to provide enforcement for security policies that you have configured. The admission controller works with validating webhooks to evaluate requests to create, update, and perform workload operations against security policies. If enforcement is configured for a policy and results in a violation, the request fails, preventing the operation from persisting in the API server and being successfully completed. 
+
+The workflow for evaluating requests and enforcing policies follows these steps:
+
+. A user or system requests to create, update, or perform workload operations that is received by the Kubernetes or {ocp} API server. 
+. The API server contacts the validating webhooks with an `AdmissionReview` request.
+. The validating webhooks call the admission controller via the service endpoint to verify that the resource being provisioned complies with the specified security policies.
+. The admission controller responds to the validating webhooks or times out. 
+* If the action will violate a security policy, the API server rejects the request.
+* If the action is not prevented by a security policy or times out, the API server accepts and persists the resource. In the case of a timeout, this behavior is known as "fail open", which is the default behavior. However, you can configure the default behavior in case of timeout to "fail closed" for stricter enforcement.
++
+[NOTE]
+====
+Before release 4.9, if a timeout or error occurred between the validating webhooks and the admission controller, the Kubernetes API would accept the request ("fail open"). Additionally, some users, for example, those using an automated CI/CD pipeline such as Argo CD, expected {product-title-short} security policy enforcement to override continuous deployment models and fail deployments that violated policy. Because the {product-title-short} validating webhook and admission controller mechanism fails open, _and_ Sensor enforcement occurs only when the deployment is created, {product-title-short} did not enforce against any later updates or scale-up events. Configuring {product-title-short} to have the API to reject the request ("fail closed") was only possible with configuration in Helm. With release 4.9, {product-title-short} provides the ability to configure if the Kubernetes API accepts the request (fails open) or rejects the request (fails closed). This setting for admission controller enforcement is initially configured on a cluster at installation and can be changed later.
```

Comment:
I would not mention a product. I would keep this generic to desired state maintenance tools.  I believe the consensus was not to add specific tools into the documentation. I understand opinions varied.

Reply:
Also, I find this section long and complex as a user, and I am not sure what the intention for it is tbh. Let's talk on a call.